lib/python/vdm/server/HTTPListener.py

```diff
@@ -2233,7 +2246,7 @@ def get(database_id):
                 try:
                     response = requests.get(url)
                 except Exception, err:
-                    return jsonify({'status': 'error', 'errorDetails': err, 'hostname': server[0]['hostname']})
+                    return jsonify({'status': 'error', 'hostname': server[0]['hostname']})
```

Comment:
This change fixes an issue with starting clusters when any node is not available. It now displays proper error message when any node is not reachable